freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClientBuilder.java

```diff
@@ -169,7 +166,7 @@ public FreeIpaClient build(boolean withPing) throws URISyntaxException, IOExcept
                 additionalHeaders.forEach(request::addHeader);
                 additionalHeadersStickySessionFirstRpc.forEach(request::addHeader);
                 try (CloseableHttpResponse response = client.execute(request)) {
-                    if (UNAVIALLBE_PING_HTTP_RESPONSES.contains(response.getStatusLine().getStatusCode())) {
+                    if (isUnreachableHttpStatus(response.getStatusLine().getStatusCode())) {
                         throw new HttpException("Ping failed with http status code " + response.getStatusLine().getStatusCode());
```

Comment:
Do you want to change LOGGER.debug(msg, e); to LOGGER.error(msg, e);  to help kibana search?


            } catch (Exception e) {
                String msg = "Ping failed";
                LOGGER.debug(msg, e);
                throw new RetryableFreeIpaClientException(msg, new FreeIpaHostNotAvailableException(msg, e));

Reply:
This error is rethrown a few times but it does get logged at the higher level with the error level.

Reply:
This is rethrown a few times and then it gets logged at the higher level at error level.